CHANGES.md

```diff
@@ -1,3 +1,9 @@
+2.0.1 (2019-06-04)
+------------------
+
+* Zed\_char: add an `indv_combining` option to the transforming functions(`of_uChars, zChars_of_uChars, of_utf8`) to determine whether extract individual combining marks from the parameter (#18)
```

Comment:
Oops! Just realized there's a small problem here. This needs to be no more than 80 columns wide! Just reformat it.

Reply:
Thank you, pmetzger. This sentence is CPL 80 formatted now.